package.json

```diff
@@ -321,19 +322,25 @@
 		"watch": "tsc -watch -p ./",
 		"pretest": "npm run compile && npm run lint",
 		"lint": "eslint src --ext ts",
-		"test": "node ./out/test/runTest.js"
+		"test": "node ./out/test/runTest.js",
+		"postinstall": "npm run download-api",
```

Comment:
The *.d.ts need not be checked in, based on the package.json, they will be pulled down automatically via the vscode-dts package. (also as the proposed API changes, the relevant *.d.ts will be automatically pulled down)